Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -4,7 +4,16 @@ import Library
 import Prelude
 import UIKit
 
+internal protocol DiscoveryPostcardCellDelegate: class {
+  func discoveryPostcardTappedShared(shareContext: ShareContext)
```

Comment:
could we add a lil comment to this to the effect of `"Called when the share button is tapped"`.